doc/api/process.md

```diff
@@ -1651,7 +1651,9 @@ added: v0.2.0
 * {Object}
 
 The `process.versions` property returns an object listing the version strings of
-Node.js and its dependencies.
+Node.js and its dependencies. `process.versions.modules` indicates the current
+ABI version, which is increased whenever a C++ API changes. Node.js will refuse
```

Comment:
I think this should be more of a "In addition, the `modules` property indicates ...", because this version contradicts the earlier sentence, the modules property is not a version of Node.js or its dependencies.

Perhaps it would be even better by enumerating each individual property, and describing it?